### ISSUE TITLE
Sorting minecraft:player_head

### ISSUE THREAD

**Issue by kzenmatthias** [yentheo/spectra-chest#1]
When filtering a chest full of different minecraft:player_head, all the items get turned into the same item and get stacked.

**Comment by yentheo** [comment]
Hey kzenmatthias! Thanks for showing interest in this mod/plugin. I have investigated this problem and it is more that the _more mobe heads_ plugin and the _player heads_ plugin don't use different "item types" for different heads. 

I've changed the implementation to be very simple at the moment. 
1. I take everything out of your inventory and put it in a temporary invisible inventory. 
2. I sort everything based on item type then on amount of items
3. And then put every stack 1 by 1 back into your inventory, it is like shift clicking every item stack into your inventory

A downside to this is that I can't choose in which inventory slot to put which stack, so for the player inventory, it is gonna use the first available slot (could be the hot bar, but I still exclude the hot bar for the first step.)

But I'm gonna change this again when I add the ability to sort in specific sections of your chest, like row per row or column per column.
